webapp/src/components/VIP/VIPInfo.tsx

```diff
@@ -202,7 +202,7 @@ const VIPInfo: React.FC = () => {
           <ActionButton
             className="btn"
             color={colors.primaryText}
-            onClick={() => window.open(URLS.vip)}
+            onClick={() => window.open(URLS.treasury)}
```

Comment:
Why is this reverted?

Reply:
VIP site is not ready yet so we are sending vips to the treasury page first.